src/index.node.js

```diff
@@ -1,5 +1,6 @@
 import { setup, setupCache, serializeQuery } from './api'
 import RedisStore from './redis'
+import RedisDefaultStore from './redis-default'
```

Comment:
we might need to rethink this class's name. can you make some other suggestions for names?

Reply:
`RedisStoreWithDefaultCommands`
??

Reply:
`RedisAlternateStore`

Existing `RedisStore` is still the default (else it will be a breaking change for those who bump their version) and this store's name shouldn't cause confusion